cloud: Update the DM maintenance (#16909)
pingcap/docs: tidb-cloud/notification-2024-04-18-dm-feature-maintenance.md

## Impact

During the maintenance window, the DM feature for TiDB Dedicated clusters in the following region will be upgraded:

- Cloud provider: AWS, region: Tokyo (ap-northeast-1)

> During the upgrade, you can use the functionalities of TiDB clusters normally. The DM tasks can keep running without disruption. The DM console can be used normally.

Reviewer: ```suggestion
During the upgrade, you can use the functionalities of TiDB clusters and the DM console normally. The DM tasks can keep running without disruption.
```
